### ISSUE TITLE
pretrain_detector

### ISSUE THREAD

**Issue by MitraTj** [rowanz/neural-motifs#64]
Hi there,

After running "bash pretrain_detector.sh", how can I find the best checkpoint to train the main model based on that pre-trained weights?

Thanks!

**Comment by rowanz** [comment]
Use whichever gives you the best validation performance, or the last one :)

**Comment by MitraTj** [comment]
Got it! Thanks!

**Comment by sharifza** [comment]
Can you report an approximation of accuracy we should expect _pretrain_detector_ to get when the model has converged enough? 

For example after 2 epochs I get:
```
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.061
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.151
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.035
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.026
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.075
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.121
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.169
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.170
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.084
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.201

```
